LCA_pipeline_setup.sh

```diff
@@ -23,7 +23,9 @@ memgb="48"
 # Include Sars-cov2 genome:
 incl_sarscov2=false
 # Working path
-work_path="."
+#work_path="."
+work_path="/share/data2/000-10x-HCA-bams/HCA_test/"
+script_path="/home/becavin/scRNAseq_pipelines/"
```

Comment:
Are you sure it's a good idea to put your own directory structure into the general script for running the pipeline?

Reply:
Hello Malte,
Nop, it is really not what we should do :) It was just to show Lisa what I meant. I sent her an email to explain why I put that. We should not hard coded it, but put-it as a parameter of the script. This will help to separate the path where the script are and the path where the data are.

Reply:
agreed!